src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
Just to confirm @mikeller , which did you prefer for the start of the scale (min or warning)?